core_ui/lib/screens/file_picker_screen/file_picker_screen.dart

```diff
@@ -17,12 +19,12 @@ class FilePickerScreen extends StatelessWidget {
       Uint8List bytes = result.files.first.bytes == null
           ? await File(result.files.first.path!).readAsBytes()
           : result.files.first.bytes!;
-      String rawContent = utf8.decode(bytes);
 
       navigatorState.push(
         MaterialPageRoute(
           builder: (_) => RenderScreen(
-            rawContent: rawContent,
+            defaultFaces:
+                ObjParser().parseContent(utf8.decode(bytes).split('\n')),
```

Comment:
I am just curious, why can't we put ObjParser as a parameter in FilePickerScreen constructor and use this parameter here?